packages/paste-website/src/pages/components/side-panel/index.mdx

```diff
@@ -64,6 +65,9 @@ export const SidePanelExample = (): React.ReactNode => {
           <Separator orientation="horizontal" verticalSpacing="space0" />
           Side Panel content goes here.
         </SidePanelBody>
+        <SidePanelFooter>
+          Chat composer goes here.
```

Comment:
question: Are these examples meant to say this? Or do they need to be replaced with the actual component?

Reply:
I'd say you could just make the text generic like "Footer content goes here" since the Body content is generic now.

(I didn't realize initially that the body content wasn't the AI Chat Log anymore)

Reply:
👍 yeah it was all just taking up too much space